docs/content/integrations/fivetran.mdx

```diff
@@ -0,0 +1,209 @@
+
+We will supply our credentials as environment variables:
+
+```bash
+export FIVETRAN_API_KEY=...
+export FIVETRAN_API_SECRET=...
+```
+
+Then, we can instruct Dagster to authorize the Fivetran resource using the environment variables:
+
+```python startafter=start_define_instance endbefore=end_define_instance file=/integrations/fivetran/fivetran.py dedent=4
+from dagster_fivetran import fivetran_resource
+
+# Pull API key and secret from environment variables
+fivetran_instance = fivetran_resource.configured(
+    {
+        "api_key": {"env": "FIVETRAN_API_KEY"},
+        "api_secret": {"env": "FIVETRAN_API_SECRET"},
+    }
+)
+```
+
+For more information on the additional configuration options available for the Fivetran resource, see the [API reference](/\_apidocs/libraries/dagster-fivetran#dagster_fivetran.fivetran_resource).
+
+---
+
+## Step 2: Loading Fivetran asset definitions into Dagster
+
+The easiest way to get started using Fivetran with Dagster is to have Dagster automatically generate asset defintions from your Fivetran project. You can [load asset definitions from a Fivetran instance via API](#loading-fivetran-asset-definitions-from-a-fivetran-instance), at initialization time.
+
+You can also [manually-build asset definitions](#manually-building-fivetran-asset-definitions) on a per-connector basis.
+
+<TabGroup>
+
+<TabItem name="Loading from Fivetran instance">
+
+### Loading Fivetran asset definitions from a Fivetran instance
+
+To load Fivetran assets into Dagster from your Fivetran instance, you will need to supply the Fivetran resource that we defined above in step 1. Here, the Fivetran instance is treated as the source of truth.
+
+```python startafter=start_load_assets_from_fivetran_instance endbefore=end_load_assets_from_fivetran_instance file=/integrations/fivetran/fivetran.py dedent=4
+from dagster_fivetran import load_assets_from_fivetran_instance
+
+# Use the fivetran_instance resource we defined in Step 1
+fivetran_assets = load_assets_from_fivetran_instance(fivetran_instance)
+```
+
+The `load_assets_from_fivetran_instance` function retrieves all of the connectors you have defined in the Fivetran interface, creating software-defined assets for each generated table. Each connector has an associated [op](https://docs.dagster.io/concepts/ops-jobs-graphs/ops#ops) which triggers a sync of that connector.
+
+</TabItem>
+<TabItem name="Manually building assets">
+
+### Manually building Fivetran asset definitions
+
+Instead of having Dagster automatically create the asset defintions for your Fivetran instance, you can opt to individually build them. First, determine the Connector IDs for each of the connectors you would like to build assets for. The Connector ID can be found on the Setup page for each connector in the Fivetran UI:
+
+<center>
```

Comment:
This image is super stretchy when viewed in preview ([link](https://dagster-git-benpankow-fivetran-docs-elementl.vercel.app/integrations/fivetran#manually-building-fivetran-asset-definitions)) - I'm not sure if it's the source image that's like this or if the pixel sizes here are incorrect. Might need to re-run `make mdx-format`

Reply:
Fixed!